packages/dependency-extraction-webpack-plugin/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Breaking
```

Comment:
Nit:

```suggestion
### Breaking Changes
```

Context: this is how we pick version bump:

https://github.com/WordPress/gutenberg/blob/73af69b59b44f28ed4abf0f8de2fdd285fd80d96/bin/plugin/commands/common.js#L66-L70